src/main/java/org/sagebionetworks/web/client/widget/entity/EntityPageTop.java

```diff
@@ -234,16 +247,69 @@ public void onSuccess(EntityBundle bundle) {
 				String wikiId = getWikiPageId(wikiAreaToken, projectBundle.getRootWikiId());
 				controller.configure(actionMenu, projectBundle, true, wikiId, entityUpdateHandler);
 				configureCurrentAreaTab();
+				showSelectedTabs();
 			}
 			
+			
+
 			@Override
 			public void onFailure(Throwable caught) {
 				projectBundleLoadError = caught;
 				configureCurrentAreaTab();
+				showSelectedTabs();
 			}
 		};
 		synapseClient.getEntityBundle(projectHeader.getId(), mask, callback);
     }
+    
+    private void showSelectedTabs() {
+		synapseClient.getCountsForTabs(entity, new AsyncCallback<ProjectDisplayBundle>() {
```

Comment:
Use projectHeader.getId() instead of entity (entity could be of any entity type).